forte/orbital-helpers/aosubspace.h

```diff
@@ -137,48 +142,29 @@ class AOSubspace {
     // Constructor with list of subspaces
     AOSubspace(std::vector<std::string> subspace_str, std::shared_ptr<psi::Molecule> molecule,
                std::shared_ptr<psi::BasisSet> basis);
+    /// Constructor using the atom normals
+    /// an atom normal is a 3D vector onto which p orbitals are projected:
+    /// px, py, pz (3 orbitals) -> nx px + ny py + nz pz (1 orbital)
```

Comment:
This function needs to document the input better. What are the two integers passed in the pair? Here you should mention that this is an optional value that can be used to define a subspace of p orbitals aligned to the atom normals.

Reply:
This is in favor of how `atom_to_aos_` is accessed. I added more comments.